test/test_mps.py

```diff
@@ -652,6 +652,7 @@ def mps_ops_modifier(ops):
         'to': None,
         'to_sparse': None,
         'unique': None,
+        'unravel_index': None,
```

Comment:
I added this to avoid the CI failure for mps, but I'm still getting the failure

Reply:
perhaps add it to MACOS_BEFORE_13_3_XFAILLIST?

Reply:
but yeah, this list should also work... odd... @kulinseth any idea?

Reply:
nvm, the issue is that here you are saying that it doesn't work for any dtype, but it's passing for int16. You should pout it in one of the XFAILLISTS (not 100% sure which one) and just xfail the relevant dtypes

Reply:
Right, i think you can move it to below [to](https://github.com/pytorch/pytorch/pull/110580/files#diff-fdee6c1352305cfc875c3f14dc43440e2c681f8f229ef8977289bb7fbe389966R707) and set the dtypes which are expected to fail.